src/stream/src/task/barrier_manager/progress.rs

```diff
@@ -42,27 +44,27 @@ impl Display for BackfillState {
 impl LocalBarrierWorker {
     pub(crate) fn update_create_mview_progress(
         &mut self,
-        current_epoch: u64,
+        epoch: EpochPair,
         actor: ActorId,
         state: BackfillState,
     ) {
-        self.state
-            .create_mview_progress
-            .entry(current_epoch)
-            .or_default()
-            .insert(actor, state);
+        if let Some(actor_state) = self.state.actor_states.get(&actor)
+            && let Some((partial_graph_id, _)) = actor_state.inflight_barriers.get(&epoch.prev)
+            && let Some(graph_state) = self.state.graph_states.get_mut(partial_graph_id)
```

Comment:
Is it possible to have `None` returned for any one of this check? Should we have a log in such cases?